### PR TITLE
Adding ability to filter out #shorts from the video lists

### PR DESCRIPTION
**Pull request by mintsoft**
Defaults to "off" so the current experience is unchanged

Fixes: https://github.com/mintsoft/kodi.plugin.ytchannels/issues/28

### CODE REVIEW

File: resources/language/resource.language.ru_ru/strings.po [hunk 1]
```diff
@@ -180,3 +180,7 @@ msgstr "Ключ YouTube API"
 msgctxt "#30208"
 msgid "Enable live streams"
 msgstr "Разрешить прямые эфиры"
+
+msgctxt "#30209"
+msgid "Filter out shorts"
+msgstr "Отфильтровать #shorts"
```
Comment (mintsoft) [inline resources/language/resource.language.ru_ru/strings.po:186]: @sryze Can you provide a better translation?

File: resources/language/resource.language.pl_pl/strings.po [hunk 2]
```diff
@@ -180,3 +180,7 @@ msgstr "Klucz API YouTube"
 msgctxt "#30208"
 msgid "Enable live streams"
 msgstr "Włącz transmisje na żywo"
+
+msgctxt "#30209"
+msgid "Filter out shorts"
+msgstr "Odfiltruj #szorty"
```
Comment (mintsoft) [inline resources/language/resource.language.pl_pl/strings.po:186]: @hevet Can you provide a better translation?

Comment (hevet) [reply resources/language/resource.language.pl_pl/strings.po:186]: @mintsoft Odfiltruj krótkie.

File: resources/language/resource.language.ru_ru/strings.po (hunk 1 above)
Comment (sryze) [reply resources/language/resource.language.ru_ru/strings.po:186]: Скрывать короткие видео

**Pull request merged**